src/site/artifacts/build.rs

```diff
@@ -0,0 +1,67 @@
+use crate::config::Config;
+use crate::errors::*;
+use crate::site::layout;
+use axohtml::{html, text, unsafe_text};
+use cargo_dist_schema::ArtifactKind;
+
+use super::fetch_manifest;
```

Comment:
might be good to reorg these imports so that similar things are next to each other- i would also prefer to not use super- maybe that function belongs in this file?

Reply:
done everywhere.

Also moved the fetch manifest to the cargo dist file